lib/top/mdx_top.ml

```diff
@@ -661,7 +726,8 @@ let init ~verbose:v ~silent:s ~verbose_findlib () =
   Compmisc.init_path true;
   Toploop.toplevel_env := Compmisc.initial_env ();
   Sys.interactive := false;
-  patch_env ();
+  (* CR hheuzard: fixme *)
+  (* patch_env (); *)
```

Comment:
What is happening here? I think this should be uncommented.

Reply:
You are right; https://github.com/realworldocaml/mdx/pull/121/commits/6856c56cd39074ea53288861ea283b867bca8209 restores the call.